Update install guide to cover PSP.enabled flag for K8s v1.25 (#567)
rancher/rancher-docs: docs/pages-for-subheaders/install-upgrade-on-a-kubernetes-cluster.md

- Set the `hostname` to the DNS name you pointed at your load balancer.
- Set the `bootstrapPassword` to something unique for the `admin` user.
- To install a specific Rancher version, use the `--version` flag, example: `--version 2.7.0`
  

> :::note
> 
> If installing on Kubernetes >= v1.25, you must add '--set global.cattle.psp.enabled=false' to your Helm install command
> 
> :::  

Reviewer: ```suggestion
- For Kubernetes v1.25 or later, set `global.cattle.psp.enabled` to `false'.
```

I think this is more appropriate as a bullet point.

- `letsEncrypt.email` is set to the email address used for communication about your certificate (for example, expiry notices)
- Set `letsEncrypt.ingress.class` to whatever your ingress controller is, e.g., `traefik`, `nginx`, `haproxy`, etc.

> :::note
> 
> If installing on Kubernetes >= v1.25, you must add '--set global.cattle.psp.enabled=false' to your Helm install command
> 
> :::
>   

Reviewer: ```suggestion
- For Kubernetes v1.25 or later, set `global.cattle.psp.enabled` to `false'.
  
```

- Set the `bootstrapPassword` to something unique for the `admin` user.
- Set `ingress.tls.source` to `secret`.

> :::note
> 
> If installing on Kubernetes >= v1.25, you must add '--set global.cattle.psp.enabled=false' to your Helm install command
> 
> :::
> 

Reviewer: ```suggestion
- For Kubernetes v1.25 or later, set `global.cattle.psp.enabled` to `false'.

```
